EveripediaNetwork/ep-api
# Pm2 server deploy

# Improve server deploy with pm2

_Make the server start along with the Indexer service restarting and watching out for the blockchain. Basically now they both start along with pm2._

## How should this be tested?

1. run pm2 restart ecosystem.config.js 
2. run pm2 logs
3. check if the applications are running correctly

## Notes or observations

_I'm having a problem with pm2 on the dev server we have. I can not set it up there to run `pm2 restart ecosystem.config.js`, even after `yarn add global pm2` went well with no errors it says that pm2 command was not found. Is it going to run correctly when deploy? I wanted to test it first._

## Linked issues

closes #00, closes #001, closes https://github.com/EveripediaNetwork/issues/issues/123
